app/assets/stylesheets/authors.css.scss

```diff
@@ -0,0 +1,13 @@
+.author {
+  clear: both;
+  .author_books {
```

Comment:
@Alexneisc Some time ago you told me that underscore in css class is a bad idea... :)